ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -410,23 +410,24 @@ private static void runStartupScripts()
     
     for (String file : files)
     {
-      try
-      {
         String language = ScriptRunner.getLanguageForFilename(file);
-        try
-        {
-          String script = loadFile(new File(STARTUP_SCRIPT_DIRECTORY, file));
-          ScriptRunner.executeScript(language, script, variables);
-        }
-        catch (IOException ex)
+        if (language != null) {
+            try 
+            {
+              String script = loadFile(new File(STARTUP_SCRIPT_DIRECTORY, file));
+              ScriptRunner.executeScript(language, script, variables);
+            }
+            catch (IOException ex)
+            {
+              ex.printStackTrace();
+            }
+          }
+        else 
         {
-          ex.printStackTrace();
+            // TODO translate this
+            // TODO use some debugging tools instead of System.err
```

Comment:
We have #50 open as a tracking issue re:proper logging. It's on my list for after #274 is resolved.

Overall, I think we'll get more mileage out of having an issue open for, say, "Translate remaining hardcoded strings" than we will from TODO comments - unless you intend to resolve the comment as we fine-tune the PR.

Reply:
That's what my TODO comments are for - resolving them as the PR evolves. And I did translate this in my working copy, was waiting for feedback before adding other commits to the PR.